server/requirements.txt

```diff
@@ -12,7 +12,7 @@ flask-restful>=0.3.9
 flask-sqlalchemy
 gunicorn
 humanize
-pquisby
+pquisby<0.0.13
```

Comment:
There appears to be a `0.0.14` available...but it looks like it is still broken.  😢

Reply:
Yeah, it's been up for a while; and, yes, it's still broken (in the same way).